packages/frontend/src/notebook/landing_page.css

```diff
@@ -0,0 +1,43 @@
+.overlay {
```

Comment:
I think this should be renamed to  `walkthrough_overlay.css`, since there is no landing page component it's not obvious from the file name what this file relates to.